capstone/src/main/webapp/script.js

```diff
@@ -12,11 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+window.addEventListener('DOMContentLoaded', (event) => {
+  if (document.getElementById('map')) {
+    createMap();
+  }
+});
+
+var map;
```

Comment:
could we put these variables inside the function?

Reply:
these vars are used in other functions, but I'll reduce the scope of the vars

Reply:
oh i misread, mb